daemon/main.c

```diff
@@ -400,11 +400,26 @@ static const char *verify_required_directory(const char *dir) {
 #ifdef ENABLE_HTTPS
 static void security_init(){
     char filename[FILENAME_MAX + 1];
-    snprintfz(filename, FILENAME_MAX, "%s/ssl/key.pem",netdata_configured_user_config_dir);
-    security_key    = config_get(CONFIG_SECTION_WEB, "ssl key",  filename);
+    char common[8];
+    snprintfz(common, 7, "none");
+    security_key    = config_get(CONFIG_SECTION_WEB, "ssl key",  common);
+    if (!strcmp(security_key, common)) {
+        snprintfz(filename, FILENAME_MAX, "%s/ssl/key.pem",netdata_configured_user_config_dir);
+        security_key    = config_get(CONFIG_SECTION_WEB, "tls key",  filename);
+    }
+
+    security_cert    = config_get(CONFIG_SECTION_WEB, "ssl certificate",  common);
+    if (!strcmp(security_cert, common)) {
+        snprintfz(filename, FILENAME_MAX, "%s/ssl/cert.pem",netdata_configured_user_config_dir);
+        security_cert    = config_get(CONFIG_SECTION_WEB, "tls certificate",  filename);
+    }
 
-    snprintfz(filename, FILENAME_MAX, "%s/ssl/cert.pem",netdata_configured_user_config_dir);
-    security_cert    = config_get(CONFIG_SECTION_WEB, "ssl certificate",  filename);
+#if OPENSSL_VERSION_NUMBER >= 0x10100000L
```

Comment:
replace `0x10100000L` with `OPENSSL_VERSION_WITH_TLS_13`

Reply:
Thank you, I missed this number!